php/php.editor/test/unit/data/goldenfiles/org/netbeans/modules/php/editor/lexer/PHP53FeaturesTest/lexer/nowdoc_008.pass

```diff
@@ -9,23 +9,25 @@ token #7 WHITESPACE \n\n
 token #8 PHP_PRINT print
 token #9 WHITESPACE  
 token #10 PHP_NOWDOC_TAG_START <<<'ENDOFNOWDOC'\n
-token #11 PHP_NOWDOC_TAG_END ENDOFNOWDOC;\n
```

Comment:
It seems that the unit test was incorrect.

Reply:
Hm, OK :)